services/core-api/app/api/mines/permits/permit_amendment/resources/permit_amendment.py

```diff
@@ -211,10 +211,12 @@ def create_standard_conditions(application_identity):
             populate_with_conditions = data.get('populate_with_conditions', True)
             if application_identity.now_application:
                 if populate_with_conditions:
-                    if application_identity.application_type_code == "ADA":
-
+                    if application_identity.application_type_code in ("ADA", "NOW"):
+                        permit_amendment_id = PermitAmendment.find_last_by_permit_id(
```

Comment:
`if application_identity.application_type_code in ("ADA", "NOW"):`

this check is no longer needed, These are the only types an application can be, this will always be true.

Reply:
Perfect, I am deleting this check. I will include this change in the next push.